ddtrace/tracer/option.go

```diff
@@ -660,7 +664,7 @@ func loadAgentFeatures(agentDisabled bool, agentURL *url.URL, httpClient *http.C
 	}
 	defer resp.Body.Close()
 	type agentConfig struct {
-		defaultEnv string `json:"default_env"`
+		DefaultEnv string `json:"default_env"`
```

Comment:
NB - This was previously broken; as the `json` package cannot do anything with non-exported fields...

Reply:
Good catch.